package.json

```diff
@@ -265,7 +265,7 @@
   "private": true,
   "dependencies": {
     "@babel/runtime": "^7.15.4",
-    "@department-of-veterans-affairs/component-library": "^11.2.2",
+    "@department-of-veterans-affairs/component-library": "11.2.6",
```

Comment:
I removed the caret and set the version to `11.2.6` so that we can [get some fixes](https://github.com/department-of-veterans-affairs/component-library/releases/tag/v11.2.6), but the package won't resolve to `11.3.*`. There are some changes in 11.3 that break cypress changes and we want to unblock a team with fixes from 11.2.6 as soon as we can.

We will work to change the cypress tests or release a new version so that we can bump the package again.